docs/PullPayments.md

```diff
@@ -0,0 +1,123 @@
+# Pull payments
+
+## Introduction
+
+Traditionally, to make a Bitcoin payment, a receiver shares their bitcoin address and the sender later sends money to this address.
+Such system is called `Push payment` as the sender initiates the payment while the receiver may be unavailable, in effect `pushing` the payment to the receiver.
+
+However, what about reversing the role?
+What if, instead of a sender `pushing` the payment, the sender allows the receiver to `pull` the payment at a time the receiver seems fit?
+This is the concept of `Pull payment`. This allows several new applications such as:
+
+* A subscription service (where the subscriber allows the service to pull money every month)
+* Refunds (where the merchant allows the customer to pull the refund money to his wallet when it seems fit)
+* Time based billing freelancing (where the person hiring allows the freelance to pull money to his wallet as time get reported)
+* Patronage (where the patron allows the recipient to pull money every month to continue supporting his work)
+* Automatic selling (where a customer of an exchange would allow an exchange to pull money from his wallet to automatically sell every month)
```

Comment:
```suggestion
* Automatic selling (where a customer of an exchange would allow an exchange to pull money from their wallet to automatically sell every month)
```